src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -379,4 +379,8 @@
   <data name="VaryByQueryKeys_Requires_ResponseCachingMiddleware" xml:space="preserve">
     <value>'{0}' requires the response cache middleware.</value>
   </data>
+  <data name="CandidateResolver_DifferentCasedReference" xml:space="preserve">
+    <value>Duplicate entry for library reference {0} found. A common cause for this issue is difference in casings for the same package identifier in different project.json files.</value>
```

Comment:
Slightly tweaked wording:

`A duplicate entry for library reference {0} was found. Please check that all package references in all projects use the same casing for the same package references.`

(I removed the mention of project.json because that's going away! 😄 )